api/docs/v2/storefront/index.yaml

```diff
@@ -137,12 +167,13 @@ components:
           type: array
           items:
             type: object
-            oneOf:
+            anyOf:
```

Comment:
It should be `oneOf` since anyOf will take all fields from all components to be accepted as far as I understand https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/